src/helm-chart/values.yaml

```diff
@@ -0,0 +1,21 @@
+name: papermc-server
+namespace: default
+
+container:
+  image: djaytan/papermc-server:dev
+  port: 25565
```

Comment:
We will never bother modifying this.
So you should remove it completely and just hardcode that is the deployment manifest.
What is important is the possibility to customize the port exposed by the service.